public/app/features/trails/ActionTabs/LayoutSwitcher.tsx

```diff
@@ -1,36 +1,43 @@
 import React from 'react';
 
 import { SelectableValue } from '@grafana/data';
-import { SceneComponentProps, SceneObject, SceneObjectBase, SceneObjectState } from '@grafana/scenes';
+import { SceneComponentProps, sceneGraph, SceneObject, SceneObjectBase, SceneObjectState } from '@grafana/scenes';
 import { Field, RadioButtonGroup } from '@grafana/ui';
 
+import { MetricScene } from '../MetricScene';
+
 export interface LayoutSwitcherState extends SceneObjectState {
-  active: LayoutType;
   layouts: SceneObject[];
   options: Array<SelectableValue<LayoutType>>;
 }
 
 export type LayoutType = 'single' | 'grid' | 'rows';
 
 export class LayoutSwitcher extends SceneObjectBase<LayoutSwitcherState> {
+  private getMetricScene() {
+    return sceneGraph.getAncestor(this, MetricScene);
+  }
+
   public Selector({ model }: { model: LayoutSwitcher }) {
-    const { active, options } = model.useState();
+    const { options } = model.useState();
+    const { layout } = model.getMetricScene().useState();
 
     return (
       <Field label="View">
-        <RadioButtonGroup options={options} value={active} onChange={model.onLayoutChange} />
+        <RadioButtonGroup options={options} value={layout} onChange={model.onLayoutChange} />
       </Field>
     );
   }
 
   public onLayoutChange = (active: LayoutType) => {
-    this.setState({ active });
+    this.getMetricScene().setState({ layout: active });
```

Comment:
would be cool to store in local storage so we can remember this as a default preference